### PR TITLE
lnbits: update app to 0.9.4

### PR DESCRIPTION
**Pull request by louneskmt**
- lnbits: update app to 0.9.4
- lnbits: update registry

### CODE REVIEW

**Comment by louneskmt** [comment]
Tested and working on a testnet amd64 Umbrel.

**Comment by leesalminen** [comment]
Also working for me on a mainnet arm64 Umbrel.

File: lnbits/docker-compose.yml [hunk 1]
```diff
@@ -7,8 +7,7 @@ services:
       APP_PORT: $APP_LNBITS_PORT
       PROXY_AUTH_ADD: "false"
   web:
-    image: lnbitsdocker/lnbits-legend:0.8.0@sha256:3426e326bac347d09b4e018780cb1ecd8e5eac35851383d476076769ab4a9b2e
-    user: 1000:1000
```
Comment (nevets963) [inline lnbits/docker-compose.yml:11]: @louneskmt Was there some reason that this was dropped?

Comment (louneskmt) [reply lnbits/docker-compose.yml:11]: They now use poetry to run the application, and it looks like it needs root permissions to be used.

**Review by nevets963 — APPROVED** [review]
Thank you @louneskmt for the LNbits app update!

Tested on `arm64` and `amd64`. Approved! 🚀 

Merging now.

**Pull request merged**